Makefile

```diff
@@ -2,7 +2,10 @@
 setup:
 	python setup.py develop
 
-test:
+clean:
+	find . -name "*.pyc" -exec rm {} +
```

Comment:
You also wouldn't want to delete their swp files or any other temporary files created by text editors.